src/odemis/gui/model/__init__.py

```diff
@@ -595,6 +597,46 @@ def __init__(self, main):
         # VA for autofocus procedure mode
         self.autofocus_active = BooleanVA(False)
 
+
+class LocalizationGUIData(MicroscopyGUIData):
+    """ Represent an interface used to only show the current data from the microscope.
+
+    It it used for handling CryoSECOM systems.
+
+    """
+
+    def __init__(self, main):
+        if main.role != "cryo-secom":
+            raise ValueError("The role of the microscope is not correct")
+        MicroscopyGUIData.__init__(self, main)
+
+        # Current tool selected (from the toolbar)
+        tools = {TOOL_NONE, TOOL_RULER}  # TOOL_ZOOM, TOOL_ROI}
+        # Update the tool selection with the new tool list
+        self.tool.choices = tools
+        # VA for autofocus procedure mode
+        self.autofocus_active = BooleanVA(False)
+        # the relative focus position below the current position
+        self.zMin = model.FloatContinuous(
+            value=-10e-6, range=(-1000e-6, 0), unit="m")
```

Comment:
I believe the comment should be the zstack minimum range below current focus position.

Reply:
maximum or minimum _range_ indicates interval. But zMax and zMin are not intervals. They are scalars, and they are measured relative to the current focus position.

Reply:
In the specification: 

> Zmin and Zmax define the range below and above the current focus position (Z) to acquire.

Both define the min and max values for the range of acquisition. The range is indeed an interval where we'll take images by zsteps from the minimum value till we reach the maximum.